src/components/SubCategory.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Entry from './Entry';
+
+
+const StyledSubCategory = styled.div`
+  width: 80%;
+
+  .subCategory-header {
+    display: flex;
+    font-size: 13px;
+    color: #4d4d4d;
+    margin: 16px;
+    font-weight: 600;
+  }
+
+  .subCategory-total {
+    margin-left: auto;
+  }
+
+  .subCategory-entries {
+    display: flex;
+    flex-wrap: wrap;
+    margin-left: 10px;
+  }
+`;
+
+const subCategoryInfo = {
+    id: 'unique',
+    title: 'Books/Movies/Music',
+  }
+      
+const subCategoryEntries = [
+  {title: "Mistborn", amount: 9.99},
+  {title: "Name of the Wind", amount: 24.95},
+  {title: "The Dark Knight", amount: 8.99},
+  {title: "The Marvelous Mrs. Maisel", amount: 4.92},
+  {title: "Mumford & Sons Vinyl", amount: 15.65}
+];
+
+const subCategoryTotal = subCategoryEntries.reduce(function(accumulator, currentValue) {
```

Comment:
**OPTIONAL** If you're comfortable with arrow functions, this can be written as

```javascript
reduce( (accumulator, currentValue) => accumulator + currentValue.amount )
```

its common to use an arrow function in array iterators, and since we're not doing any computational logic or creating variables, we can just return the single statement. ROT (rule of thumb): If we have no new variables, and are returning a single statement, we can skip the `{}` block. Same goes for the `.map` below.

Good practice note, when naming your iterator variables (e.g. currentValue) it's helpful to name it the thing you expect it to be, in this case `entry`. Later when you come back to this function you might forget what `currentValue` represents, and why it has an `amount` field. 

Up to you if you want to leave the code as is, it functions perfectly well. just QOL (quality of life)